sipXyealink/etc/yealink/types.xml

```diff
@@ -200,7 +201,7 @@
 </type>
 <type id="InfoType_type">
     <enum>
-	<option><label>Disabled</label><value>0</value></option>
+	<!--<option><label>Disabled</label><value>0</value></option>-->
```

Comment:
Sounds good, we could define a new <type id="InfoType_type_v8"> and refer it in v8 xml config files